patient_summaries.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import warnings
+import pandas as pd
+import numpy as np
+import labkey
+from labkey_client import SMMARTLabkey
+import json
+
+# Command line arguments:
+#     --id  Create a summary table for a specific patient in addition to the summary of all the patients. Default is 101.
+#     --out_dir  Where to save the output file. Default is Desktop.
+# Additional markers and genes can be included by adding them to the appropriate json config dictionary. Any value that exists in the LabKey tables should work.
+# Two important options are 'cnv_filter' and 'mutation_filter'. Both are set to 'true' by default, so the LabKey query will only select records when the 'Reported' has a true value.
+# It is assumed the combination of 'Participant ID' and 'Date/Collection Date' can be used as a key when merging/joining records from different tables. 
+# 'Bems IDs' would be useful for this, but unfortunately the 'Bems ID' is often missing for a record. 
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--id', default = 0, type = int,  help = 'Create a summary table for a specific patient in addition to the summary of all the patients.')
+    parser.add_argument('--study', default = 'amtec', type = str,  help = 'Create a summary table for patients in a spcefic study.')
+    parser.add_argument('--out_dir', default = '~/Desktop', type = str, help = 'Where to save the output file.')
+    args = parser.parse_args()
+
+    # Use to join tables
+    INDEX = ['Participant ID', 'Date']
+
+    with open ('patient_summaries_config.json') as json_config:
```

Comment:
Recommend combining arguments and config such that:
1. all current args can be specified in the config
2. user can either specify args or provide config